packages/dso-toolkit/components/Componenten/progress-indicator/progress-indicator.njk

```diff
@@ -1,3 +1,3 @@
-<div {{ className('dso-progress-indicator', 'dso-progress-indicator-looping', [size, 'dso-' + size]) }} role="progressbar" aria-valuetext="{{ status }}">
+<div {{ className('dso-progress-indicator', 'dso-progress-indicator-looping', [size, 'dso-' + size], [block, 'dso-block'], [block, 'dso-' + color]) }} role="progressbar" aria-valuetext="{{ status }}">
```

Comment:
`[block, 'dso-' + color]` ? klopt dat?

Reply:
Geen kleur zonder dat het een "block" is, dat waren toch de specs? Anders zou je ook een linksuitgelijnd blok met achtergrondkleur zonder padding kunnen maken.

Reply:
Ja, maar wat gebeurt er nu dan als `block` `true` is, maar er geen `color` bestaat? Ik ken nunjucks niet zo goed...

Reply:
Jaja ik zie het al, dan krijg je `dso-undefined`.